index.js

```diff
@@ -24,15 +24,15 @@ module.exports = function (obj, opts) {
 	};
 
 	var seen = [];
-	return (function stringify(parent, key, node, level) {
-		var indent = space ? '\n' + new Array(level + 1).join(space) : '';
+	return (function stringify(parent, key, node, path = []) {
```

Comment:
using this syntax would be a breaking change.

Reply:
```suggestion
	return (function stringify(parent, key, node, path = []) {
	  path = typeof(path) === 'number' ? new Array(path) : path;
```

Reply:
the breaking syntax is default parameters - `=` in a function signature.

Reply:
cool then
```suggestion
	return (function stringify(parent, key, node, path = []) {
	      path = typeof(path) === 'number' ? new Array(path) : (path == null ? [] : path);
```

Reply:
again, it's the `path = []` in the signature that can't work.